hudi-utilities/src/test/java/org/apache/hudi/utilities/deltastreamer/TestHoodieDeltaStreamer.java

```diff
@@ -1851,42 +1851,66 @@ public void testNullSchemaProvider() {
   }
 
   @Test
-  public void testPayloadClassUpdate() throws Exception {
-    String dataSetBasePath = basePath + "/test_dataset_mor_payload_class_update";
+  public void testPartialPayloadClass() throws Exception {
```

Comment:
nit: the original name still works?  Also should we only keep one test with parameterization on table types by merging `testPartialPayloadClass` (on MOR) and `testPayloadClassUpdateWithCOWTable` (on COW)?